### PR TITLE
MueLu: add cmake guard if Galeri disabled

### PR DESCRIPTION
**Pull request by jhux2**
Fixes issue #5595

---
This change is [Reviewable](https://reviewable.io/reviews/trilinos/trilinos/5601)

### CODE REVIEW

File: packages/muelu/example/basic/CMakeLists.txt [hunk 1]
```diff
@@ -5,7 +5,7 @@ INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR})
 #
 
 IF(HAVE_MUELU_INST_DOUBLE_INT_INT)
```
Comment (mhoemmen) [inline packages/muelu/example/basic/CMakeLists.txt]: Y'all realize that this test will never get enabled in the future, right?

Comment (lucbv) [reply packages/muelu/example/basic/CMakeLists.txt]: @mhoemmen we won't allow `int_int` only `long_int_int` and `long_long_int_int` ?

Comment (mhoemmen) [reply packages/muelu/example/basic/CMakeLists.txt]: That's correct, but this test only gets enabled for `int_int` right?

**Comment by tjfulle** [comment]
@jhu, autotester reports conflicts in your changes.  Did someone else modify this file too?

**Comment by jhux2** [comment]
This appears to be fixed in develop already.  I'm going to delete this PR.

**Pull request closed without merging**